mule-apikit-tools/mule-apikit-scaffolder/src/main/java/org/mule/tools/apikit/Scaffolder.java

```diff
@@ -208,7 +208,6 @@ public void run() {
     try {
       muleConfigGenerator.generate();
       muleArtifactJsonGenerator.generate();
-      scaffolderReport.setStatus(ScaffolderReport.SUCCESS);
```

Comment:
Why removing this fixes the issue? Is the idea that the default report status is `success` and only overwritten when it fails?

Reply:
As discussed with @juanchib, yes, that's the idea